src/mixins/name-request-mixin.ts

```diff
@@ -24,7 +24,10 @@ export default class NameRequestMixin extends Vue {
    * @returns the name request object
    */
   async fetchValidateNameRequest (nrNumber: string, phone?: string, email?: string): Promise<NameRequestIF> {
-    const nameRequest = await LegalServices.fetchNameRequest(nrNumber, phone, email).catch(error => {
+    let nameRequest: NameRequestIF
+    try {
+      nameRequest = await LegalServices.fetchNameRequest(nrNumber, phone, email)
+    } catch(error: any) {
```

Comment:
I needed to do this because I mock fetchNameRequest and the mock implementation apparently doesn't support `.catch()` function.